x/authz/client/cli/tx.go

```diff
@@ -53,8 +57,10 @@ func GetTxCmd() *cobra.Command {
 
 func NewCmdGrantAuthorization() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "grant <grantee> <authorization_type=\"send\"|\"generic\"|\"delegate\"|\"unbond\"|\"redelegate\"> --from <granter>",
-		Short: "Grant authorization to an address",
+		Use:        "grant <grantee> <authorization_type=\"send\"|\"generic\"|\"delegate\"|\"unbond\"|\"redelegate\"> --from <granter>",
```

Comment:
I would just use `'` quotes instead of escaping. Makes it easier to read and shorter.